src/FiniteVolumeUpscale.hpp

```diff
@@ -57,7 +57,7 @@ class FiniteVolumeUpscale : public Upscale
     FiniteVolumeUpscale(MPI_Comm comm,
                         const mfem::SparseMatrix& vertex_edge,
                         const mfem::Vector& weight,
-                        const mfem::Array<int>& global_partitioning,
+                        const mfem::Array<int>& partitioning,
```

Comment:
Yes, `partitioning` here is for the local graphs of individual processors, while `global_partitioning` in `GraphUpscale` is for the global graph before distribution. 

This is because of the setup of `ParGraph`, which needs to be changed (now the global graph is partitioned and distributed based on global partition, it should be better to distribute first and then partition local graphs). But I plan to change it in this PR.